src/main/java/com/checkconsulting/proepargne/repository/CollaboratorRepository.java

```diff
@@ -2,12 +2,16 @@
 
 import com.checkconsulting.proepargne.model.Collaborator;
 import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
 import org.springframework.stereotype.Repository;
 
+import java.util.List;
 import java.util.Optional;
 
 @Repository
 public interface CollaboratorRepository extends JpaRepository<Collaborator, Long> {
 
     Optional<Collaborator> findByEmail(String email);
+    @Query(value = "select c from Collaborator c where c in ( select a.collaborator from Account a where a.contract = ( select c from Contract c where c.companyAdminId = ?1))")
+    List<Collaborator> findAllCompanyAdminCollaborators(String adminId);
 }
```

Comment:
a-t-on vraiment besoin d'une custom query ici, pouvons-nous pas lier les collaborateurs à l'employé par l'ajout d'un champ one To Many, si ça n'existe pas déjà

Reply:
Il n'y a pas besoin de faire de jointure ni de relation comme pour le contrat. On a besoin de récupérer l'ID admin et ajouter une colonne.